tests/test_pycdf.py

```diff
@@ -690,8 +693,8 @@ class MakeCDF(unittest.TestCase):
     def setUp(self):
         self.testdir = tempfile.mkdtemp()
         self.testfspec = os.path.join(self.testdir, 'foo.cdf')
-        pth = os.path.dirname(os.path.abspath(__file__))
-        self.testmaster = os.path.join(pth, 'po_l1_cam_testc.cdf')
+        self.testmaster = os.path.join(spacepy_testing.testsdir,
```

Comment:
Is there any particular reason to keep the test cdfs in the `tests` directory rather than `tests/data`?

Reply:
> Is there any particular reason to keep the test cdfs in the `tests` directory rather than `tests/data`?

Because they were the first test data we had and I wasn't organized enough to have a "data" directory yet. (Actually it's probably because they were the first _tests_ and predate the inclusion of pycdf in spacepy.) I'm happy to move them but probably out of scope for this PR; I can open another.